test/factories/factories.rb

```diff
@@ -8,9 +8,13 @@
     "ILikeKumqutas#{n}@gmail.com"
   end
 
+  sequence :first_name do |n|
+    "Roger#{n}"
+  end
+
   factory :user do
     username "LambPETAsAreTasty"
-    first_name  "Roger"
+    first_name
```

Comment:
There is a first name still. It's in a sequence above the test because I thought I might be reusing first_name in other factories... so I abstracted it. I was incorrect so I can just put the sequence back inline instead of abstracted.